Sources/App/Commands/Analyze.swift

```diff
@@ -634,15 +634,18 @@ func dumpPackage(at path: String) throws -> Manifest {
         // up the tree through parent directories to find one
         throw AppError.invalidRevision(nil, "no Package.swift")
     }
-    let swiftCommand = Current.fileManager.fileExists("/swift-5.4/usr/bin/swift")
-        ? "/swift-5.4/usr/bin/swift"
-        : "swift"
-    let json = try Current.shell.run(command: .init(string: "\(swiftCommand) package dump-package"),
-                                     at: path)
+    let json = try Current.shell.run(command: .swiftDumpPackage, at: path)
```

Comment:
This was outdated - we've not been including a 5.4 version since 5.3 was the main Swift version we built the server with.